### PR TITLE
Change web root in config general

### PR DESCRIPTION
**Pull request by fernandog**
@medariox @OmgImAlexis 

also log the web root in the log

### CODE REVIEW

File: medusa/server/core.py [hunk 1]
```diff
@@ -202,8 +202,11 @@ def run(self):
             protocol = 'http'
             self.server = HTTPServer(self.app)
 
-        logger.log('Starting Medusa on {scheme}://{host}:{port}/'.format
-                   (scheme=protocol, host=self.options['host'], port=self.options['port']))
+        logger.log('Starting Medusa on {scheme}://{host}:{port}{web_root}'.format
```
Comment (OmgImAlexis) [inline medusa/server/core.py]: Does this not need `/` before and after the `{web_root}` or does `web_root` include a starting and ending one?

File: views/config_general.mako [hunk 2]
```diff
@@ -458,6 +458,19 @@
...
                         </div>
+                        <div class="field-pair">
+                            <label for="web_root">
+                                <span class="component-title">HTTP web root</span>
+                                <span class="component-desc">
+                                    <input type="text" name="web_root" id="web_root" value="${app.WEB_ROOT}" class="form-control input-sm input300" autocomplete="no" />
+                                    blank = disabled
+                                <div class="clear-left">
+                                <p>Set a base URL, for use in reverse proxies.</p>
+                                <p><b>Note:</b> Must restart to have effect. Mind that old URL will not work.</p>
```
Comment (OmgImAlexis) [inline views/config_general.mako]: "Mind that old URL will not work."

This needs to be reworded.

File: medusa/server/core.py (hunk 1 above)
Comment (fernandog) [inline medusa/server/core.py]: @OmgImAlexis medusa auto add the `/` to web_root

File: views/config_general.mako (hunk 2 above)
Comment (duramato) [inline views/config_general.mako]: "Keep in mind that any previously configured base URLs won't work, after this change."

Comment (fernandog) [inline views/config_general.mako]: suggestion?

File: medusa/server/core.py (hunk 1 above)
Comment (fernandog) [inline medusa/server/core.py]: `INFO     TORNADO :: [d958c98] Starting Medusa on http://0.0.0.0:8081/medusa`

```python
        # web root
        if self.options['web_root']:
            app.WEB_ROOT = self.options['web_root'] = ('/' + self.options['web_root'].lstrip('/').strip('/')
```

Comment (OmgImAlexis) [inline medusa/server/core.py]: All good, just thought I'd check. 👍

**Comment by fernandog** [comment]
@medariox any comments?

**Pull request merged**